README.md

```diff
@@ -17,14 +17,16 @@ If you are looking for the source code of the [React Native Archive website](htt
 ### Prerequisites
 
 1. [Git](https://git-scm.com/downloads).
-1. [Node](https://nodejs.org/en/download/) _(version 12 or greater)_.
-1. [Yarn](https://yarnpkg.com/lang/en/docs/install/) _(version 1.5 or greater)_.
+1. [Node](https://nodejs.org/en/download/) _(>=v20)_.
+1. [Yarn](https://yarnpkg.com/corepack/) _(v4)_.
```

Comment:
```suggestion
1. [Node](https://nodejs.org/en/download/) _(version 20 or greater)_..
1. [Yarn](https://yarnpkg.com/corepack/) _(version 4 or greater)_.
```

There is no need to abbreviate the message, let's retain the existing form, just replace with correct values.